MFVNeutralino/plugins/TriggerEfficiency.cc

```diff
@@ -276,30 +321,87 @@ void MFVTriggerEfficiency::analyze(const edm::Event& event, const edm::EventSetu
     }
   }
 
+  if (require_metfilters) {
+    if ( ! triggerfloats->pass_metfilters ){
+      return;
+    }
+  }
+  double Wmass = 0;
   if (require_muon) {
     edm::Handle<pat::MuonCollection> muons;
     event.getByToken(muons_token, muons);
 
     int nmuons[2] = {0};
+    double mu_pt = -999;
+    double mu_phi = -999;
     for (const pat::Muon& muon : *muons) {
       for (int i = 0; i < 2; ++i) {
-        if (i == 0 || muon_selector(muon)) {
+        //if (i == 0 || (muon_selector(muon) && muon.passed(reco::Muon::CutBasedIdTight))) { //&& muon.passed(reco::Muon::CutBasedIdTight)
+        if (i == 0 || (muon.passed(reco::Muon::CutBasedIdTight) && muon_selector(muon))) {
```

Comment:
Are both of these needed? `muon.passed(reco::Muon::CutBasedIdTight) && muon_selector(muon)`

Reply:
I don't think both are needed. I just did that to avoid differences.

Reply:
Okay! I think we should just use the muon.passed version--if (accidentally) they aren't identical, then we might inadvertently be applying a selection that is tighter than one or both of them. Shouldn't matter much for our trigger efficiency studies of course, but it would if we relied more heavily on these objects!